app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -91,6 +91,17 @@ def link_to_field(name, value, label = nil, facet_hash = {})
       link_to(label, main_app.search_catalog_path(state))
     end
 
+    def index_facet_link options={}
```

Comment:
@jcoyne I misunderstood. It's all new to me, so we'll double back and review the `link_to_search` and the code which we generated for the index links in the tests. Also we'll need to do something about the nested `span`